README.md

```diff
@@ -11,225 +11,165 @@ If you are a JIRA admin user, you can configure a JIRA WebHook (as shown in the
-    <entity name="user" type="com.symphony.integration.jira.user" version="1.0">
-      <attribute name="username" type="org.symphonyoss.string" value="test" />
-      <attribute name="emailAddress" type="org.symphonyoss.string" value="test@symphony.com" />
-      <attribute name="displayName" type="org.symphonyoss.string" value="Test User" />
-    </entity>
-    <entity type="com.symphony.integration.jira.labels" version="1.0">
-      <attribute name="production" type="org.symphonyoss.string" value="production" />
-    </entity>
-    <entity type="com.symphony.integration.jira.issue" version="1.0">
-      <attribute name="project" type="org.symphonyoss.string" value="Sample 1" />
-      <attribute name="key" type="org.symphonyoss.string" value="SAM-24" />
-      <attribute name="subject" type="org.symphonyoss.string" value="Sample Bug Blocker" />
-      <attribute name="type" type="org.symphonyoss.string" value="Bug" />
-      <attribute name="link" type="com.symphony.uri" value="https://whiteam1.atlassian.net/browse/SAM-24" />
-      <attribute name="priority" type="org.symphonyoss.string" value="Highest" />
-      <attribute name="status" type="org.symphonyoss.string" value="Done" />
-      <entity name="assignee" type="com.symphony.integration.jira.user" version="1.0">
-        <attribute name="username" type="org.symphonyoss.string" value="test2" />
-        <attribute name="emailAddress" type="org.symphonyoss.string" value="test2@symphony.com" />
-        <attribute name="displayName" type="org.symphonyoss.string" value="Test2 User" />
-      </entity>
-    </entity>
-    <entity type="com.symphony.integration.jira.changelog" version="1.0">
-      <entity type="com.symphony.integration.jira.change" version="1.0">
-        <attribute name="fieldName" type="org.symphonyoss.string" value="resolution" />
-        <attribute name="oldValue" type="org.symphonyoss.string" value="null" />
-        <attribute name="newValue" type="org.symphonyoss.string" value="Done" />
-      </entity>
-      <entity type="com.symphony.integration.jira.change" version="1.0">
-        <attribute name="fieldName" type="org.symphonyoss.string" value="status" />
-        <attribute name="oldValue" type="org.symphonyoss.string" value="To Do" />
-        <attribute name="newValue" type="org.symphonyoss.string" value="Done" />
-      </entity>
-    </entity>
-  </entity>
-</messageML>
-```
-* Message rendered on Symphony (since this sample does not have a "description" tag, it will show as "n/a")
-
-![Issue Updated rendered](src/docs/sample/sample_issue_updated_rendered.png)
-
-### Comment Created
 
-* [Comment Created JSON file](src/docs/sample/jiraCallbackSampleCommentAdded.json)
-* Message ML generated by the parsers
+##### MessageML and JSON entity supported by the Universal Webhook Integration (MessageML v2)
+###### MessageML
```

Comment:
You should add further information about MessageML v2 like Agent version that supports MessageML v2.

Example: https://github.com/symphonyoss/App-Integrations-Zapier#generated-symphony-message-and-json-entity-messageml-v2